### ISSUE TITLE
Add to Edge (Chromium) Store

### ISSUE THREAD

**Issue by andre-paulo98** [igrigorik/videospeed#588]
Edge has released on a Chromium Version, you can install it through the Google Chrome store, but it would be better to have a version uploaded on Edge Store.
https://microsoftedge.microsoft.com/addons/category/Edge-Extensions

I've tested the extension on Edge and seems to be working correctly.

**Comment by ChadBailey** [comment]
I like the suggestion, as I see more exposure as a good thing. Not sure if I can agree with the "better" statement though lol. This is something only @igrigorik could make happen though, so would have to wait and see how he feels about it.

**Comment by snowman** [comment]
see #335

**Comment by TheZoker** [comment]
The difference is, that the other issue talks about the legacy Edge version and this issue is about the chromium Edge version. So two different browsers

**Comment by ChadBailey** [comment]
I think I'm going to go ahead and take this one on, to see if I can enhance/extend the browser support has part of #963

**Comment by ChadBailey** [comment]
Just to limit the scope of my activities I've temporarily descoped this until the refactored version gets merged. I'll take a second look once everything is merged over, as firefox is going to be a bit more complex.

Keeping this open so it's not forgotten about.
